server/src/main/java/org/folio/reservoir/server/UploadService.java

```diff
@@ -138,15 +136,15 @@ private Future<Void> uploadContent(ReadStream<Buffer> request,
     return uploadPayloadStream(parser, ingestWriteStream, queueSize);
   }
 
-  private String enforcePermissionsBySource(RoutingContext ctx, String sourceId) {
+  private void enforcePermissionsBySource(RoutingContext ctx, String sourceId) {
```

Comment:
@adamdickmeiss The reason why this returns a string is to make sure that all return paths are exhausted. Plus, It would be better if this was changed in a separate PR.